ambry-api/src/main/java/com/github/ambry/config/MySqlAccountServiceConfig.java

```diff
@@ -97,6 +98,13 @@
   @Default("false")
   public final boolean updateDisabled;
 
+  /**
+   * If true, MySqlAccountService would ignore version mismatch while updating Accounts and Containers.
+   */
+  @Config(IGNORE_VERSION_MISMATCH)
+  @Default("false")
+  public final boolean ignoreVersionMismatch;
```

Comment:
I would move this to base AccountServiceConfig class so AbstractAccountService can also use it.